recipes/unifrac/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "unifrac" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/biocore/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 97507caa5536bf445ca712ec495a39c4c95db3388d3259be3270e0e15ceb5ce5 
+
+build:
+  number: 0
+  skip: True  # [py<35 or win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - setuptools
+    - numpy >=1.12.1
+    - scikit-bio >=0.5.1
+    - Cython >=0.28.3
+    - hdf5 >=1.8.17
+    - h5py
+    - biom-format
+  host:
+    - python
+    - pip
+    - numpy >=1.12.1
+    - Cython >=0.28.3
+  run:
+    - scikit-bio >=0.5.1
```

I presume you need Python at run time?
```suggestion
    - python
    - scikit-bio >=0.5.1
```